.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

```suggestion
    branches: [ master ]
  pull_request:
    branches: [ master ]
```

they are created with this name in this repo